MusicApp/DataBase/Repositories/AlbumRepository.cs

```diff
@@ -0,0 +1,98 @@
+﻿using DataBase.Models;
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Utils;
+
+namespace DataBase.Repositories;
+
+public static class AlbumRepository
+{
+    public static int GetAlbumsCount()
+    {
+        using (ApplicationContext db = new ApplicationContext())
+        {
+            return db.Albums
+                .Count();
+        }
+    }
+    public static List<AlbumModel> GetAlbumsForPagination(DefaultPagination @params)
+    {
+        using (ApplicationContext db = new ApplicationContext())
+        {
+            return db.Albums
+                .Where(x => x.Genres.Contains(@params.Genre)) // || @params.Genre == null)
+                .OrderBy(x => @params.SortBy)
+                .ThenBy(x => @params.OrderBy)
+                .Skip((@params.PageNumber - 1) * @params.PageSize)
+                .Take(@params.PageSize)
+                .ToList();
+        }
+    }
+    public static List<AlbumModel> GetAllAlbums(string id)
```

Comment:
better to specify which id it is -- `artistId`

Reply:
Done